eoscostarica/eos-rate
# feat(demux): add bpjson crawling and storing support for all bps

### GH Issue

#148

### Steps to test

1. If you have `eoslocal` running, stop it.
1. If you already had data stored in the DB about block producers, please execute `make flush` to get a brand new copy.
1. Run this project with `make start`.
1. After launching, you should be able to see all of the block producers along with their proper bpjson files attached.

This PR has demux and data cleaner deactivated for now until we sort out our eosrate contract.

### Checklist
